qemu/tests/slof_greater_lun_id.py

```diff
@@ -0,0 +1,57 @@
+"""
+slof_greater_lun_id.py include following case:
+ 1.SLOF could support LUN ID greater than 255.
+"""
+
+import logging
+
+from virttest import error_context
+from provider import slof
+from virttest import utils_net
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify SLOF info when LUN ID greater than 255.
+
+    Step:
+     1. Boot a guest with scsi system disk which lun=0
+     2. Check no any errors from output of slof.
+     3. Could login guest.
+     4. Could ping external host ip.
+     5. Shutdown guest.
+     6. Change the lun id of scsi disk greater than 255(e.g 300).
+     7. Boot this guest again.
+     8. Repeat to do step 2 ~ 4.
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    params['start_vm'] = 'yes'
+    start_pos = 0
+    for params['drive_lun_image1'] in (
+            params['boot_lun_id'], params['reboot_lun_id']):
+        env_process.preprocess_vm(test, params, env, params["main_vm"])
+        vm = env.get_vm(params["main_vm"])
+        vm.verify_alive()
+        content, next_pos = slof.wait_for_loaded(vm, test, start_pos)
+
+        error_context.context("Check the output of SLOF.", logging.info)
+        slof.check_error(test, content)
+
+        error_context.context("Try to log into guest '%s'." % vm.name,
+                              logging.info)
+        timeout = float(params.get("login_timeout", 240))
+        session = vm.wait_for_login(timeout=timeout)
+        logging.info("log into guest '%s' successfully." % vm.name)
+
+        error_context.context("Try to ping external host.", logging.info)
+        extra_host_ip = utils_net.get_host_ip_address(params)
+        session.cmd_status_output('ping %s -c 5' % extra_host_ip)
```

Comment:
Why not check the command result status here?

Reply:
thanks your comment, have updated.